src/models/participantEndpoint/facade.js

```diff
@@ -26,8 +26,8 @@
 
 const Logger = require('@mojaloop/central-services-shared').Logger
 const Cache = require('./participantEndpoint')
```

Comment:
I dont believe that the facade is used anywhere. please verify and remove if necessary